advisories/unreviewed/2022/05/GHSA-x534-j49x-mqvj/GHSA-x534-j49x-mqvj.json

```diff
@@ -1,19 +1,43 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-x534-j49x-mqvj",
-  "modified": "2023-03-01T18:30:58Z",
+  "modified": "2023-03-09T05:05:46Z",
   "published": "2022-05-24T16:57:50Z",
   "aliases": [
     "CVE-2019-11932"
   ],
-  "details": "A double free vulnerability in the DDGifSlurp function in decoding.c in libpl_droidsonroids_gif before 1.2.15, as used in WhatsApp for Android before 2.19.244, allows remote attackers to execute arbitrary code or cause a denial of service.",
+  "summary": "Double free in android-gif-drawable",
+  "details": "A double free vulnerability in the DDGifSlurp function in decoding.c in the android-gif-drawable library before version 1.2.18, as used in WhatsApp for Android before version 2.19.244 and many other Android applications, allows remote attackers to execute arbitrary code or cause a denial of service when the library is used to parse a specially crafted GIF image.",
   "severity": [
     {
       "type": "CVSS_V3",
       "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H"
     }
   ],
-  "affected": [],
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "Maven",
+        "name": "pl.droidsonroids.gif:android-gif-drawable"
```

Comment:
Is this enough for Dependabot to notice this, and for it to be synced to https://osv.dev/vulnerability/CVE-2019-11932?

If not, could you please review this advisory so it goes from "unreviewed" to "reviewed"?
According to the presentation https://media.ccc.de/v/38c3-ultrawide-archaeology-on-android-native-libraries#t=1244 a lot of Android apps might still be using vulnerable versions.
Though maybe should be done together with #5164 then so users directly update to >= 1.2.24.

Reply:
> and for it to be synced to https://osv.dev/vulnerability/CVE-2019-11932

Probably has no effect on osv.dev unfortunately, see https://github.com/google/osv.dev/discussions/3049.